src/DataStructures/Tensor/Expressions/AddSubtract.hpp

```diff
@@ -71,6 +71,9 @@ struct AddSub<T1, T2, ArgsList1<Args1...>, ArgsList2<Args2...>, Sign>
       "S^b_a, which doesn't make sense. The indices may also be in different "
       "frames, different types (spatial vs. spacetime) or of different "
       "dimension.");
+  static_assert(Sign == 1 or Sign == -1,
+                "Invalid Sign provided for addition or subtraction of Tensor "
+                "elements. Sign must be 1 (addition) or -1 (subtraction).");
```

Comment:
Here is a rough draft of an assert to handle the case when an invalid value for `Sign` is given.

Reply:
Looks good!